INSTALL.md

```diff
@@ -170,6 +170,15 @@ especially if you're using a user-managed node install such as [nvm](https://git
     Windows, check the [FAQ item "Docker seems to be
     unavailable"](https://github.com/balena-io/balena-cli/blob/master/TROUBLESHOOTING.md#docker-seems-to-be-unavailable-error-when-using-windows-subsystem-for-linux-wsl).
 
+* The `balena scan` command require the **Bonjour** service.
+  * macOS comes with [Bonjour](http://www.apple.com/macosx/features/bonjour/) built-in
```

Comment:
* require → requires
* The link doesn't work for me (http://www.apple.com/macosx/features/bonjour/). It redirects to 'support.apple.com'. Perhaps link to Wikipedia instead: https://en.wikipedia.org/wiki/Bonjour_(software)